proxy/proxy.go

```diff
@@ -437,7 +437,12 @@ func sendDirectResponse(w http.ResponseWriter, r *http.Request,
 		w.Header().Set(hdrGrpcStatus, strconv.Itoa(int(codes.Internal)))
 		w.Header().Set(hdrGrpcMessage, errInfo)
 
-		w.WriteHeader(statusCode)
+		// As per the gRPC spec, we need to send a 200 OK status code
```

Comment:
Based on this: https://github.com/grpc/grpc/blob/master/doc/http-grpc-status-mapping.md

Should the above instead use an `UNKNOWN` status/error code?

Reply:
Yes, probably. Or we could even send our own `Grpc-Status` code (for example 402). But because all clients out there explicitly check for the `codes.Internal` code at the moment, we would need to upgrade all clients first. Or parse the `User-Agent` field which contains the version of the `grpc` library. But that just seemed too brittle to me.

Reply:
Gotcha, let's stick with this for now then. When we refine the bLIP we can make sure to note these aspects for backwards compat.